src/components/SocialMediaLinks/SocialMediaLinks.test.tsx

```diff
@@ -0,0 +1,243 @@
+// I have no idea why, so I am just going to use the array directly each time.
+// const smlData = [
+//   { labelText: "Alt Twitter", type: "twitter", url: "twitter.com/elsewhere", },
+//   { labelText: "NYPL Facebook", type: "facebook", url: 'facebook.com/nypl', },
+// ];
+
+describe("SocialMediaLinks Accessibility", () => {
+  describe("Labels", () => {
+    it("passes axe accessibility test without visible labels", async () => {
+      const { container } = render(<SocialMediaLinks />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with labels set to true", async () => {
+      const { container } = render(<SocialMediaLinks showLabels={true} />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+  describe("Colors", () => {
+    it("passes axe accessibility test with link color & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} color={"link"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with textInverse color & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} color={"link"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+  describe("Borders", () => {
+    it("passes axe accessibility test with straight borders & labels", async () => {
+      const { container } = render(
+        <SocialMediaLinks showLabels={true} border={"straight"} />
+      );
+      expect(await axe(container)).toHaveNoViolations();
+    });
+    it("passes axe accessibility test with circular borders", async () => {
+      const { container } = render(<SocialMediaLinks border={"circular"} />);
+      expect(await axe(container)).toHaveNoViolations();
+    });
+  });
+});
+
+describe("SocialMediaLinks", () => {
+  it("Renders selected tags", () => {
+    render(
+      <SocialMediaLinks
+        layout={"column"}
+        linksData={[
+          {
+            labelText: "Alt Twitter",
+            type: "twitter",
+            url: "twitter.com/elsewhere",
+          },
+          {
+            labelText: "NYPL Facebook",
+            type: "facebook",
+            url: "facebook.com/nypl",
+          },
+        ]}
+      />
+    );
+
+    // @todo In general, how can you be sure that the thing that matches the role is not a false positive?
+    // @todo There could be two links, for example, to other things. How do you target these queries?
+
+    // @todo Do we need to test if the icons are in a row or column?
+
+    // There should be two links. @todo Do we need to test if they are the correct URL?
+    expect(screen.getAllByRole("link")).toHaveLength(2);
+    // The label text we desire should be in the document. @todo Do we need to test if it is visible when it should be? Should we test that both labels ore there?
```

Comment:
The unit test does not know css so testing whether it is visible or not is difficult -- to be in the document is enough.
You can test both labels are there.